### PR TITLE
Pd 619 backup naming scheme

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

File: content/shared/cloud-snaps-restore-an-incremental-backup.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+hidden: true
+---
+
+To restore an incremental backup, you must restore the previous full backup and all incremental backups performed since the last full backup.
```
Comment (nathan-px) [inline content/shared/cloud-snaps-restore-an-incremental-backup.md:5]: I'm not really following this. I think we need more information. Is there a specific procedure users need to follow for this, or is it more of a note?

Comment (andreipope) [reply content/shared/cloud-snaps-restore-an-incremental-backup.md:5]: > I'm not really following this. I think we need more information. Is there a specific procedure users need to follow for this, or is it more of a note?

@nathan-px Yes, it's a note. Shall we wrap it in an `info` tag?

Comment (nathan-px) [reply content/shared/cloud-snaps-restore-an-incremental-backup.md:5]: @andreipope If it's a note for an existing task, yeah -- it should be wrapped in an `info` tag and be placed under the relevant task. If it's a new task with its own heading like we've presented here, it needs its own task documentation showing users how to restore the incremental backups.

Comment (andreipope) [reply content/shared/cloud-snaps-restore-an-incremental-backup.md]: @nathan-px It's updated.

**Review by nathan-px — APPROVED** [review]

**Pull request merged**